skfuzzy/fuzzymath/fuzzy_ops.py

```diff
@@ -661,6 +661,60 @@ def interp_membership(x, xmf, xx):
     return xxmf
 
 
+def interp_value(x, xmf, y):
```

Comment:
What do you think about renaming this `interp_universe` to make its inverse nature relative to `interp_membership` a little clearer and more intuitive?